run_scans.py

```diff
@@ -3,26 +3,25 @@
 def launch(args):
   return launchArgs(*args)
 
-def launchArgs(tool, shelf, slot, link, chamber, vfatmask, scanmin, scanmax, nevts, stepSize=1,
-               vt1=None,vt2=0,mspl=None,perchannel=False,trkdata=False,ztrim=4.0,
+def launchArgs(tool, cardName, shelf, link, vfatmask, scanmin, scanmax, nevts, startTime, stepSize,
+               vt1,vt2,mspl,l1atime,perchannel=False,trkdata=False,ztrim=4.0,
                config=False,amc13local=False,t3trig=False, randoms=0, throttle=0,
-               internal=False, debug=False):
-  import datetime,os,sys
+               internal=False, debug=False, voltageStepPulse=False, latency=33, CalPhase=0,
+               chMin=0, chMax=127, calSF=0, pulseDelay=40):
+  import os,sys
   import subprocess
   from subprocess import CalledProcessError
   from gempython.gemplotting.mapping.chamberInfo import chamber_config
   from gempython.utils.wrappers import runCommand
 
-  startTime = datetime.datetime.now().strftime("%Y.%m.%d.%H.%M")
   dataPath = os.getenv('DATA_PATH')
-
   scanType = "vt1"
   dataType = "VT1Threshold"
 
   #Build Commands
   setupCmds = []
   preCmd = None
-  cmd = ["%s"%(tool),"-s%i"%(slot),"-g%i"%(link),"--shelf=%i"%(shelf), "--nevts=%i"%(nevts), "--vfatmask=0x%x"%(vfatmask)]
+  cmd = ["%s"%(tool),"--cardName=%s"%(cardName),"-g%s"%(link),"--L1Atime=%s"%(l1atime), "--mspl=%s"%(mspl),"--nevts=%s"%(nevts), "--vfatmask=0x%x"%(vfatmask), "--pulseDelay=%s"%(pulseDelay), "--scanmin=%s"%(scanmin), "--scanmax=%s"%(scanmax), "--ztrim=%s"%(ztrim),  "--stepSize=%s"%(stepSize)]
```

Comment:
except for `cardName` all of these values are integers not strings, use `%i`.  Leave `vfatmask` alone though, that should be reported in hex.